src/components/ApartmentApplicantList/components/ApplicantListItem/index.js

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import 'date-fns';
+import {
+  Grid,
+  ListItem,
+  ListItemAvatar,
+  ListItemSecondaryAction,
+  ListItemText,
+  Avatar,
+  IconButton,
+} from '@material-ui/core';
+import PersonIcon from '@material-ui/icons/Person';
+import ClearIcon from '@material-ui/icons/Clear';
+import user from '../../../../services/user';
+import '../../../../views/ApartmentApp/apartmentApp.css';
+import '../../../../views/PeopleSearch/components/PeopleSearchResult/peopleSearchResult.css';
+
+// Based off src/views/PeopleSearch/components/PeopleSearchResult
+// but using this.props.profile of type StudentProfileInfo
+// rather than using this.props.Person of type PeopleSearchResult
+export default class ApplicantListItem extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      avatar: null,
+      prefImage: null,
+      defImage: null,
+    };
+  }
+
+  handleRemove = (profile) => {
+    // Make sure the chosen profile was not null
+    if (profile) {
+      // Send the selected profile to the parent component
+      this.props.onApplicantRemove(profile);
+    }
+  };
+
+  componentDidUpdate(newProps) {
+    if (this.props.profile.AD_Username !== newProps.profile.AD_Username) {
+      this.loadAvatar();
+    }
+  }
+
+  componentDidMount() {
+    this.loadAvatar();
+  }
+
+  async loadAvatar() {
+    this.setState({ avatar: null });
+    const [{ def: defaultImage, pref: preferredImage }] = await Promise.all([
+      await user.getImage(this.props.profile.AD_Username),
+    ]);
+    let avatar;
+    if (this.props.profile.AD_Username) {
+      avatar = preferredImage || defaultImage;
+    } else {
+      avatar = (
+        <svg width="50" height="50" viewBox="0 0 50 50">
+          <rect width="50" height="50" rx="10" ry="10" fill="#CCC" />
+        </svg>
+      );
+    }
+    this.setState({ avatar });
+  }
+
+  render() {
+    const profile = this.props.profile;
+    let fullName = String(profile.fullName);
+    let personType, personClassJobTitle, nickname;
+
+    // set nicknames up
+    if (
+      profile.NickName !== null &&
+      profile.NickName !== '' &&
+      profile.FirstName !== profile.NickName
+    ) {
+      nickname = '(' + profile.NickName + ')';
+    }
```

Comment:
this line 80 takes the user's nickname (if they have one) and encapsulates it in parentheses, so that it may be displayed as "firstname (nickname) lastname ", like on the student profile page
For example, it would display you as "Nicholas (Nick) Noormand" instead of "Nicholas Nick Noormand "